ee/tables/homebrew/upgradeable.go

```diff
@@ -74,12 +75,24 @@ func (t *Table) generate(ctx context.Context, queryContext table.QueryContext) (
 	for _, dataQuery := range tablehelpers.GetConstraints(queryContext, "query", tablehelpers.WithDefaults("*")) {
 		// Brew can take a while to load the first time the command is ran, so leaving 60 seconds for the timeout here.
 		var output bytes.Buffer
-		if err := tablehelpers.Run(ctx, t.slogger, 60, allowedcmd.Brew, []string{"outdated", "--json"}, &output, &output, tablehelpers.WithUid(uid)); err != nil {
+		var stderr bytes.Buffer
+
+		targetCmd := allowedcmd.Brew
+		cmdArgs := []string{"outdated", "--json"}
+
+		// override cmd and args for darwin because we must run disclaimed
+		if runtime.GOOS == "darwin" {
+			targetCmd = allowedcmd.Launcher
+			cmdArgs = []string{"rundisclaimed", "brew", "outdated", "--json"}
+		}
```

Comment:
I wonder if we can bake this into `tablehelpers.Run`

Reply:
I added a `tablehelpers.Disclaimed` for reusability within Run, let me know what you think!